apps/web/src/views/Swap/V3Swap/containers/SwapCommitButton.tsx

```diff
@@ -46,14 +50,24 @@ interface SwapCommitButtonPropsType {
   tradeLoading?: boolean
 }
 
+export function useMaxAmountIn(trade: SmartRouterTrade<TradeType>, slippage: Percent, amountIn = trade?.inputAmount) {
```

Comment:
what's this for? I think we already have `useSlippageAdjustedAmounts` hooks for slippage adjustment?